setup.cfg

```diff
@@ -32,7 +32,7 @@ console_scripts =
 [options.extras_require]
 testing =
     black==23.1.0
-    chex>=0.1.5
+    chex>=0.1.5,<=0.1.6
```

Comment:
And this one because newer versions of chex have slightly different type checking consequences and were throwing errors. I guess we really do need to pin down as many versions as possible if we don't want our test suites to be constantly breaking.